shell-completion/zsh/_systemd-analyze

```diff
@@ -54,6 +54,7 @@
             'timestamp:Parse a systemd syntax timestamp'
             'timespan:Parse a systemd syntax timespan'
             'security:Analyze security settings of a service'
+            'inspect-elf:Parse and print ELF package metadata'
```

Comment:
I think needs some annotation that an executable file should follow. But I don't know how to do this off the top of my head. Let's leave that for later.

Reply:
yeah, me neither, never really used zsh